tests/core/initializers_test.py

```diff
@@ -0,0 +1,189 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Tests for the Gromov Wasserstein."""
+
+import jax
+import jax.numpy as jnp
+import jax.test_util
+import numpy as np
+from absl.testing import absltest, parameterized
+
+from ott.core import initializers as init_lib
```

Comment:
For consistency with other imports, would prefer, e.g., `from ott.geometry import geometry, pointcloud` and use `pointcloud.PointCloud`.

Reply:
+1